doc/user/demos/log-parsing.md

```diff
@@ -157,11 +157,10 @@ impose a structure on them:
 
 ```sql
 CREATE SOURCE requests
-FROM 'file:///log/requests'
-WITH (
-    regex='(?P<ip>\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}) - - \[(?P<ts>[^]]+)\] "(?P<path>(?:GET /search/\?kw=(?P<search_kw>[^ ]*) HTTP/\d\.\d)|(?:GET /detail/(?P<product_detail_id>[a-zA-Z0-9]+) HTTP/\d\.\d)|(?:[^"]+))" (?P<code>\d{3}) -',
-    tail=true
-    );
+FROM FILE '/log/requests'
+WITH ( tail=true )
```

Comment:
Nikhil's Official Style Recommendation:

```sql
CREATE SOURCE requests
FROM FILE '/log/requests' WITH (tail = true)
FORMAG REGEX '...getwrecked...'
```

(at the very least I think you've got an erroneous trailing comma)

Reply:
thanks for catching the comma